packages/zowe-explorer-api/src/vscode/ui/PaginationCodeLens.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as vscode from "vscode";
+
+export class LoadMoreCodeLens implements vscode.CodeLensProvider {
```

Comment:
Wondering if we should swap some naming around. 🤔 

For example, we can name the class `PaginationCodeLens` which may allow for a more pagination related items to be exposed at a later time.